packages/webpack-plugin/src/webpack-bundle-analysis/webpackBundleAnalysisPlugin.ts

```diff
@@ -25,6 +26,10 @@ export const webpackBundleAnalysisPlugin: BundleAnalysisUploadPlugin = ({
     await output.write();
   },
   webpack(compiler) {
+    const webpack = createRequire(import.meta.url)(
```

Comment:
I generally prefer if this statement is split up to make it easier to read.

```
const generatedRequire = createRequire(import.meta.url);
const webpack = generatedRequire('webpack') as typeof TWebpack;
```

could you also sanity check that this transpiles properly for the cjs build?

Reply:
Here's what's in the `.cjs` file:

```js
    const generatedRequire = node_module.createRequire(
      typeof document === "undefined"
        ? require("u" + "rl").pathToFileURL(__filename).href
        : (_documentCurrentScript && _documentCurrentScript.src) ||
            new URL("index.cjs", document.baseURI).href,
    );
    const webpack = generatedRequire("webpack");
```

And what's in the `.mjs`:

```js
    const generatedRequire = createRequire(import.meta.url);
    const webpack = generatedRequire("webpack");
```

Reply:
Unminified*

Reply:
I believe that looks right, since the `cjs` is using `require` directly

Reply:
yup that is correct